File: DotNet/src/JustGiving.Api.Sdk/JustGivingClient.cs
using System;
using JustGiving.Api.Sdk.ApiClients;
using JustGiving.Api.Sdk.Http;
using JustGiving.Api.Sdk.Http.MicrosoftHttp;

namespace JustGiving.Api.Sdk
{
    public class JustGivingClient: JustGivingClientBase
    {
        public JustGivingClient(string apiKey)
            : base(new ClientConfiguration(apiKey), new HttpClientWrapper(), null, null, null, null, null, null)
        {
        }

        public JustGivingClient(ClientConfiguration clientConfiguration)
            : base(clientConfiguration, new HttpClientWrapper(), null, null, null, null, null, null)
        {
        }

        public JustGivingClient(ClientConfiguration clientConfiguration, IHttpClient httpClient)
            : base(clientConfiguration, httpClient, null, null, null, null, null, null)
        {
        }

        public JustGivingClient(ClientConfiguration clientConfiguration, IHttpClient httpClient, IAccountApi accountApi,
                                IDonationApi donationApi, IPageApi pageApi, ISearchApi searchApi, ICharityApi charityApi,
                                IEventApi eventApi): base(clientConfiguration, httpClient, accountApi, donationApi, pageApi, searchApi, charityApi, eventApi)
        {
        }
    }
}

File: PHP/JustGivingClient.php
<?php

include_once 'ApiClients/PageApi.php';
include_once 'ApiClients/AccountApi.php';
include_once 'ApiClients/CharityApi.php';
include_once 'ApiClients/DonationApi.php';
include_once 'ApiClients/SearchApi.php';
include_once 'ApiClients/EventApi.php';
include_once 'ApiClients/TeamApi.php';

class JustGivingClient
{	
	public $ApiKey;
	public $ApiVersion;
	public $Username;
	public $Password;
	public $RootDomain;
	
	public $Page;
	public $Account;
	public $Charity;
	public $Donation;
	public $Search;
	public $Event;
	public $Team;

	public function __construct($rootDomain, $apiKey, $apiVersion, $username="", $password="")
	{
		$this->RootDomain   	= (string) $rootDomain; 
		$this->ApiKey     		= (string) $apiKey;
		$this->ApiVersion     	= (string) $apiVersion;
		$this->Username     	= (string) $username;
		$this->Password     	= (string) $password;
		$this->curlWrapper		= new CurlWrapper();
		$this->debug			= false;
		
		// Init API clients
		$this->Page				= new PageApi($this);
		$this->Account			= new AccountApi($this);
		$this->Charity			= new CharityApi($this);
		$this->Donation			= new DonationApi($this);
		$this->Search			= new SearchApi($this);
		$this->Event			= new EventApi($this);
		$this->Team			    = new TeamApi($this);
	}
}